config/purposes/scrna_core_cell_extraction.wip.yml

```diff
@@ -52,7 +52,12 @@ LRC Blood Bank:
 # Output plate from Hamilton PBMC isolation protocol / input plate for the cell counting and cell banking protocols
 LRC PBMC Bank:
   :asset_type: plate
+  # Labware creator for pooling isolations by sample before cell counting
+  :creator_class: LabwareCreators::PooledWellsBySampleInGroups
   :label_template: plate_cellaca_qc # creates QC1 up to QC4 barcodes
+  # number of source wells to pool into a single destination well by sample from LRC Bank to LRC PBMC Bank
+  # e.g. 6 copies of sample in 6 wells, '2' in the config, pool to make 3 child wells
+  :number_of_source_wells: 2
```

Comment:
if `number_of_source_wells` is specific to the labware creator you could use the args syntax like on lines 97-102 below so it's kept together